OWNERS_ALIASES

```diff
@@ -5,12 +5,10 @@ aliases:
   # tasks on the repo, or otherwise approve any PRs.
   kubebuilder-admins:
     - pwittrock
-    - estroz
     - camilamacedo86
 
   # non-admin folks who can approve any PRs in the repo
   kubebuilder-approvers:
-    - adirio
```

Comment:
Should this be an empty list to be valid? Or is it ok like this?

Reply:
It seems ok to have empty list.

Reply:
We need more contributions @erikgb and @FillZpp by sure
I hope that up to the end of this year we will be able to grow this list
and you both are very welcome as well ;-)